### ISSUE TITLE
Does not work in Proxmox 7

### ISSUE THREAD

**Issue by Monklite** [nmaggioni/r710-fan-controller#9]
I recently updated Proxmox and tried to re-run this. I have not had any issues with this previously. The first time I ran it post-update, I got:  

```
*** Installing packaged dependencies...
Hit:1 http://ftp.us.debian.org/debian bullseye InRelease
Get:2 http://ftp.us.debian.org/debian bullseye-updates InRelease [36.8 kB]
Hit:3 http://security.debian.org bullseye-security InRelease
Hit:4 http://download.proxmox.com/debian/pve bullseye InRelease
Fetched 36.8 kB in 1s (43.7 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
#!/bin/bash
Reading state information... Done
Package python-virtualenv is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'python-virtualenv' has no installation candidate
```

So I edited the .sh file  to remove python-virtualenv as a candidate since python3-virtualenv should be used as default now and then got:

```
*** Installing packaged dependencies...
Hit:1 http://ftp.us.debian.org/debian bullseye InRelease
Hit:2 http://ftp.us.debian.org/debian bullseye-updates InRelease
Hit:3 http://security.debian.org bullseye-security InRelease
Hit:4 http://download.proxmox.com/debian/pve bullseye InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
build-essential is already the newest version (12.9).
ipmitool is already the newest version (1.8.18-10.1).
libsensors4-dev is already the newest version (1:3.6.0-7).
python3-virtualenv is already the newest version (20.4.0+ds-2).
python3-dev is already the newest version (3.9.2-3).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
*** Creating folder '/opt/fan_control'...
*** Creating and activating Python3 virtualenv...
*** Installing Python dependencies...
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: initfsencoding: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fee84932740 (most recent call first):
./install.sh: line 36: 536229 Aborted                 pip3 install -r requirements.txt
```

Any idea how to fix this?

**Comment by Monklite** [comment]
Alright so I removed the `/opt/fan_control/venv` folder and re-installed. It seems fine now... Might be a good idea to clear that out each time it is re-installed just for good measure.  

Thoughts?

**Comment by nmaggioni** [comment]
Thanks for your report, should be fixed by 4392ba1.
